### PR TITLE
Aa/groupping even tho the branch name is connection this is actually the group sorting part 

### PR DESCRIPTION
**Pull request by arminakvn**
this has the block style code + group + sorting by group if passed + the code attempts on the phase style

### CODE REVIEW

**Review by tadiraman — APPROVED** [review]

File: src/components/Block/Block.scss [hunk 1]
```diff
@@ -1,48 +1,92 @@
...
+    &-handle {
         height: 100%;
+        z-index: 2;
+        cursor: col-resize;
+        border-radius: space.$s100;
     }
 
-    &-label {
+    &-GroupLabel {
         position: absolute !important;
```
Comment (nickcaler) [inline src/components/Block/Block.scss:51]: Todo:  Go through the css in this repo and get rid of all the `!important`s. Gonna make a card from this.

File: src/components/Block/Block.tsx [hunk 2]
```diff
@@ -102,6 +102,7 @@ export default observer(function Block(props: BlockProps) {
 
     const handleWidth = {
         flex: `0 0 ${config.resizeHandleWidth}px`,
+        borderRadius: '2px'
```
Comment (nickcaler) [inline src/components/Block/Block.tsx:105]: we may want to pull this from the tokens?

File: src/components/Block/Block.tsx [hunk 3]
```diff
@@ -119,12 +120,20 @@ export default observer(function Block(props: BlockProps) {
     }
 
     return (
+        <>
+         { (blocks.groupBy && block.visible) ? <> {
+                block.groupName && 
+                    <span className='ReactTimeline__Block-GroupLabel' style={{left: `${ spaces.timeToPx(block.timespan.start)}px`, top: `${block.y - viewport.top - 25}px`, position: 'absolute'}}> 
+                        { (block.groupName !== 'nan') && block.groupName}
```
Comment (nickcaler) [inline src/components/Block/Block.tsx:127]: I think this should be
```suggestion
                        { (block.groupName.toLowerCase() !== 'nan') && block.groupName}
```

**Review by nickcaler — APPROVED** [review]
Just some minor things.  :shipit:

**Pull request merged**